recipe/meta.yaml

```diff
@@ -30,7 +32,7 @@ requirements:
     - cmake
     - ninja
   host:
-    - libboost-devel
+    - libboost-devel >=1.84
```

Comment:
You should not have this kind of pin as this will choose always the latest boost version as a pin. If you want a newer one, explicitly set it.